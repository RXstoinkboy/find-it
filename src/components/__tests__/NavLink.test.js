import React from 'react'
import NavLink from '../NavLink'
import {shallow} from 'enzyme'
import toJson from 'enzyme-to-json'

describe('NavLink', ()=>{
    it('renders correctly',()=>{
        const wrapper = shallow(<NavLink name='about us'/>);
        
        expect(toJson(wrapper)).toMatchSnapshot()
    })
})
