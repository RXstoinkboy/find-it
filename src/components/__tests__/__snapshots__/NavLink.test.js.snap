// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NavLink renders correctly 1`] = `
<div
  className="wrapper"
>
  <Link
    className="link"
    to="/about us"
  >
    about us
  </Link>
</div>
`;
